smarts/core/agent_manager.py

```diff
@@ -118,6 +119,11 @@ def pending_agent_ids(self) -> Set[str]:
         """The IDs of agents that are waiting to enter the simulation"""
         return self._pending_agent_ids
 
+    @property
+    def pending_social_agent_ids(self) -> Set[str]:
+        """The IDs of social agents that are waiting to enter the simulation"""
+        return self._pending_social_agent_ids
```

Comment:
I am unsure if I should distinguish between social agents and ego agents in this way.